packages/lab/src/tabs-next/TabNext.css

```diff
@@ -0,0 +1,122 @@
+/* Class applied to root Tab element */
+.saltTabNext {
+  --tabs-tab-background: var(--salt-navigable-primary-background);
+  --tabs-tab-background-hover: rgba(0, 0, 0);
+  --tabs-tab-backgroundOpacity-hover: 0.08;
+  --tabs-tab-color: var(--salt-text-primary-foreground);
+  --tabs-tab-letterSpacing: 0;
+  --tabs-tab-cursor: pointer;
+  --tabs-tab-position: relative;
+}
+
+.salt-theme[data-mode="dark"] .saltTabNext {
+  --tabs-tab-background-hover: rgba(255, 255, 255);
+}
+
+.saltTabNext {
+  flex-shrink: 0;
+  align-items: center;
+  align-self: stretch;
+  justify-content: center;
+  background: var(--saltTabs-tab-background, var(--tabs-tab-background));
+  border: none;
+  border-radius: 0;
+  color: var(--saltTabs-tab-color, var(--tabs-tab-color));
+  cursor: var(--saltTabs-tab-cursor, var(--tabs-tab-cursor));
+  display: var(--tabs-tabstrip-display);
+  letter-spacing: var(--saltTabs-tab-letterSpacing, var(--tabs-tab-letterSpacing));
+  min-width: var(--saltTabs-tab-minWidth, var(--salt-spacing-400));
+  outline: none;
+  position: var(--saltTabs-tab-position, var(--tabs-tab-position));
+  border-bottom-width: var(--tabs-activationIndicator-thumb-height);
+  border-bottom-color: transparent;
+  border-bottom-style: solid;
+  user-select: none;
+}
+
+.saltTabNext:focus-visible::after {
+  content: "";
+  inset: 2px 2px 2px 2px;
+  outline-color: var(--salt-focused-outlineColor);
+  outline-offset: var(--salt-focused-outlineOffset);
+  outline-style: var(--salt-focused-outlineStyle);
+  outline-width: var(--salt-focused-outlineWidth);
+  position: absolute;
+}
+
+.saltTabNext[aria-selected="true"] {
+  color: var(--salt-text-primary-foreground);
+  font-weight: var(--salt-navigable-fontWeight-active);
+}
+
+.saltTabNext-dragAway {
+  display: none;
+}
+
+/* main content aria of Tab */
+.saltTabNext-main {
+  border: none;
+  color: inherit;
+  cursor: inherit;
+  align-items: center;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+  height: var(--salt-size-base);
+  margin-top: var(--salt-spacing-50);
+  outline: none;
+  padding: 0 var(--tabs-tab-spacing);
```

Comment:
Why? The tabstrip controls the spacing

Reply:
between the tabs and inside the tab?

Reply:
since it doesn't get redefined to any other value can just use the spacing token directly

Reply:
It will be easier to change in the future if I don't have to hunt down the use of spacing tokens and instead can refactor the top level tabs variables only.